composer.json

```diff
@@ -12,7 +12,7 @@
         }
     ],
     "require": {
-        "php"                   : ">=5.5",
+        "php"                   : "^5.5.9",
```

Comment:
Interesting :) I thought long and hard about how to handle the "magic" installation of the Blackfire and New Relic extensions.

If I auto-inject them to the list of requirements (upon add-on detection), users who have those add-ons will not be able to use PHP 7, and might be confused and ask why. So I made the decision to attempt an install after the "real" app dependencies had been resolved, and emit a warning if it failed (there's also a warning on dyno start).

Also, adding a `ext-blackfire` or `ext-newrelic` requirement would mean that someone who pushes with an explicit PHP 7 requirement would get a build failure. Figuring out whether to add such a requirement is hard - that's why I'm using Composer, because it does that resolution for me! :)

I want to keep it that way, because it does not mess with the "frozen" set of dependency declarations a user made and it means builds always succeed.

I think you have two options:

 1. add `ext-blackfire` to `require`, and also add it to `config`.`platform` so that users without the extension on their computer don't get an error on install/update - this will lead to PHP 5.6 being selected on Heroku because the `blackfire` extension is taken into account on initial resolution;
 1. have a separate `heroku.composer.json` with different dependencies, and do `heroku config:set COMPOSER=heroku.composer.json` to tell Heroku to use that file on install (but then you always need to remember to maintain both).

I'd pick the first one.